### PR TITLE
doc/man: build and install cephfs-shell manpage

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
changelog

- do not install man page if `WITH_CEPHFS_SHELL` is not enabled.

**Comment by tchaikov** [comment]
https://shaman.ceph.com/builds/ceph/wip-cephfs-shell-manpage-kefu/91d2f4a1b33c6dcf5b23cf01e6d143755db6413a/

**Comment by tchaikov** [comment]
@batrick sorry for the FTBFS. should have been fixed. could you please take another look?

**Review by batrick — APPROVED** [review]

**Review by varshar16 — CHANGES_REQUESTED** [review]
This PR breaks the link to cephfs-shell doc on this [page](https://ceph--41165.org.readthedocs.build/en/41165/cephfs/#getting-started-with-cephfs). Please fix it.

**Comment by tchaikov** [comment]
@varshar16 fixed and repushed. could you please take another look?

File: doc/cephfs/index.rst [hunk 1]
```diff
@@ -57,7 +57,7 @@ for interactive access or scripting via the `cephfs-shell`_.
 .. _deploy MDS manually as needed: add-remove-mds
 .. _Orchestrator deployment table: ../mgr/orchestrator/#current-implementation-status
 .. _Mount CephFS\: Prerequisites: mount-prerequisites
-.. _cephfs-shell: cephfs-shell
+.. _cephfs-shell: ../../man/8/cephfs-shell
```
Comment (varshar16) [inline doc/cephfs/index.rst:60]: This one still needs to be fixed

**Review by varshar16 — CHANGES_REQUESTED** [review]

File: doc/cephfs/index.rst (hunk 1 above)
Comment (tchaikov) [reply doc/cephfs/index.rst]: @varshar16 should be fixed now. thanks for checking the link, Varsha.

**Review by varshar16 — APPROVED** [review]

**Pull request merged**